docs/01-Introduction.md

```diff
@@ -1,6 +1,89 @@
 # 1.0 - Introduction
 
-## 1.1 What is LORIS?
+### 1.1 What is LORIS-MRI? 
+LORIS-MRI is the backbone of the imaging component that makes up LORIS.
+ It is maintained in a separate repository so that it can be installed
+ on the file server and separated from the web server.
+These documents assume you have some knowledge regarding LORIS and
+a functioning installation. For information regarding LORIS itself,
+please consult the [LORIS wiki][1].
 
-## 1.2 How does LORIS work?
+LORIS-MRI is a set of libraries responsible for the insertion,
+organization, and archival of uploaded imaging datasets.
+ It expects an uploaded, compressed file containing
+a [DICOM][2] study composed of several DICOM files. These DICOM files
+ will be archived on the server and converted to [MINC][3] and [NIfTI][4]
+files. Knowledge of these file formats can be helpful, but are not 
+necessary for using or installing LORIS-MRI.
 
+LORIS-MRI allows you to easily organize and archive your imaging datasets
+and links them with
+ corresponding behavioral data in LORIS. Scans can be viewed and quality
+ controlled through the user's web browser, facilitating collaboration
+ between radiologists, clinicians and researchers.
+
+### 1.2 How does LORIS-MRI work?  ![user_story](images/user_story.png)
+LORIS-MRI allows multiple ways to upload scans, more details about these
+options can be found [here.](05-PipelineLaunchOptions.md) Typically, users
+upload a compressed (.tgz, .tar.gz, or .zip) DICOM folder to Imaging
+Uploader.  LORIS requires that the uploaded file name follow the naming
+convention PSCID_CANDID_VISIT-LABEL.  Assuming the upload went through
+successfully, an entry is added to the menu of the Imaging Uploader module. 
+The status of the upload during the insertion pipeline can then be tracked 
+by the user through the Log Viewer.
+
+All DICOM datasets uploaded via the Imaging Uploader or transferred on the 
+LORIS-MRI server must be free of any identifying
+information (e.g. patient name). A tool can be provided to the sites to
+facilitate deidentification. Please contact the LORIS team for details.
+
+The LORIS-MRI pipeline starts once the scans are uploaded to the server.
+The pipeline can start automatically if the autolaunch configuration is
+set, otherwise
+ manual intervention is required by a developer with backend access to
+ the server (information on the scripts needed to run the pipeline can
```

Comment:
Here, I would tend to add the link to the section about how to launch the pipeline (the link you have is useful, but too general as it related to all the scripts, and some **re**insertion scripts, not just how to launch the pipeline)...